RDBM/Integer.cpp

```diff
@@ -41,6 +41,10 @@ int& Integer::Get()
 {
 	return value;
 }
+void* Integer::Getv()
+{
+	return &value;
```

Comment:
(void*)&value.
I'm afraid of  the fact that not all compilers will let this go.